site/content/docs/5.1/components/orange-local-headers.md

```diff
@@ -0,0 +1,191 @@
+      <img src="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-320.png" alt=""/>
+    </picture>
+  </div>
+</div>
+{{</ example >}}
+
+### Breadcrumb
+
+{{< callout info >}}
+Example will come with issue [#891]({{<param repo>}}/issues/891).
+{{</ callout >}}
+
+### Local navigation bar
+
+{{< callout info >}}
+Example will come with issue [#890]({{<param repo>}}/issues/890).
+{{</ callout >}}
+
+## Title bars
+
+A title bar is designed to display the title of a page and shall be placed right below a global header.
+
+Being based on spacing and backgrounds utilities, title bars were designed to take `100vw` width.
+
+### Background color
+
+To change the appearance of a title bar, use our [background utilities]({{<docsref "/utilities/background">}}). Orange branded background utilities are `.bg-dark`, `.bg-white` and `.bg-supporting-*`.
+
+Background color other than white or black must be used only in title bars containing a **short title** and at least **one image**.
+
+{{< example class="p-0">}}
+<div class="bg-white title-bar">
+  <div class="container">
+    <h1>Title</h1>
+  </div>
+</div>
+
+<div class="mt-3 bg-light"></div>
+
+<div class="bg-dark title-bar">
+  <div class="container">
+    <h1>Title</h1>
+  </div>
+</div>
+
+<div class="mt-3 bg-light"></div>
+
+<div class="bg-supporting-pink title-bar">
+  <div class="container">
+    <h1>Title</h1>
+    <picture>
+      <source media="(min-width:1440px)" srcset="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-1440.png">
+      <source media="(min-width:1280px)" srcset="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-1280.png">
+      <source media="(min-width:1024px)" srcset="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-1024.png">
+      <source media="(min-width:768px)" srcset="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-768.png">
+      <source media="(min-width:480px)" srcset="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-480.png">
+      <source media="(min-width:320px)" srcset="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-320.png">
+      <img src="/docs/{{< param docs_version >}}/assets/img/title-bars-illustrations/illustration-320.png" alt=""/>
+    </picture>
+  </div>
+</div>
+{{</ example >}}
+
+### Images
+
+Several options are available to add an image in a title bar.
+
+#### Srcset
+
+We **strongly recommend** to use `srcset` attribute as it is very well supported by browsers and allows you to load an image depending on the device of the user. However, it might introduce a delay to display the image while resizing.
+
+{{< example class="p-0">}}
+ <div class="bg-supporting-pink title-bar">
+    <div class="container">
+      <h1>Srcset</h1>
```

Comment:
Instead of writing title representing the current section (e.g. "Srcset") I would stick with the "Title" string.